controller/space_iterations_test.go

```diff
@@ -70,87 +74,34 @@ func (rest *TestSpaceIterationREST) UnSecuredController() (*goa.Service, *SpaceI
 	return svc, NewSpaceIterationsController(svc, rest.db, rest.Configuration)
 }
 
-func (rest *TestSpaceIterationREST) TestSuccessCreateIteration() {
-	// given
-	var p *space.Space
-	var rootItr *iteration.Iteration
-	ci := createSpaceIteration("Sprint #21", nil)
-	err := application.Transactional(rest.db, func(app application.Application) error {
-		repo := app.Spaces()
-		newSpace := space.Space{
-			Name:    "TestSuccessCreateIteration" + uuid.NewV4().String(),
-			OwnerID: testsupport.TestIdentity.ID,
-		}
-		createdSpace, err := repo.Create(rest.Ctx, &newSpace)
-		p = createdSpace
-		if err != nil {
-			return err
-		}
-		// create Root iteration for above space
-		rootItr = &iteration.Iteration{
-			SpaceID: newSpace.ID,
-			Name:    newSpace.Name,
-		}
-		iterationRepo := app.Iterations()
-		err = iterationRepo.Create(rest.Ctx, rootItr)
-		return err
+func (rest *TestSpaceIterationREST) TestCreate() {
+	rest.T().Run("success", func(t *testing.T) {
+		t.Run("ok", func(t *testing.T) {
+			// given
+			ci := createSpaceIteration("Sprint #42", nil)
+			fxt := tf.NewTestFixture(t, rest.DB, tf.CreateWorkItemEnvironment())
+			svc := testsupport.ServiceAsUser("Iteration-Service", *fxt.Identities[0])
+			ctrl := NewSpaceIterationsController(svc, rest.db, rest.Configuration)
+			// when
+			resp, iter := test.CreateSpaceIterationsCreated(t, svc.Context, svc, ctrl, fxt.Spaces[0].ID, ci)
+			// then
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.playload.res.golden.json"), iter)
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok.headers.res.golden.json"), resp)
+		})
+		t.Run("with force active", func(t *testing.T) {
+			// given
+			ci := createSpaceIteration("Sprint #43", nil)
+			ci.Data.Attributes.UserActive = ptr.Bool(true)
+			fxt := tf.NewTestFixture(t, rest.DB, tf.CreateWorkItemEnvironment())
+			svc := testsupport.ServiceAsUser("Iteration-Service", *fxt.Identities[0])
+			ctrl := NewSpaceIterationsController(svc, rest.db, rest.Configuration)
+			// when
+			resp, iter := test.CreateSpaceIterationsCreated(t, svc.Context, svc, ctrl, fxt.Spaces[0].ID, ci)
+			// then
+			compareWithGoldenAgnostic(t, filepath.Join(rest.testDir, "create", "ok_with_force_active.playload.res.golden.json"), iter)
```

Comment:
`ok_with_force_active.playload.res.golden.json` 
Please correct the name of the golden file. You are using the word "playload". It should be "payload".

Reply:
@DhritiShikhar thank you. Fixed in c5b961c2be8e6d14cd5d05a9fd79d13fae360956